pkg/cmd/render/render.go

```diff
@@ -131,14 +130,12 @@ func (r *renderOpts) Validate() error {
 		return err
 	}
 
-	// TODO add after installer PR is merged
-	//if len(r.operandKubernetesVersion) == 0 {
-	//	return errors.New("missing operand kubernetes version: --operand-kubernetes-version")
-	//}
-	//if _, err := semver.Parse(r.operandKubernetesVersion); err != nil {
-	//	return err
-	//}
-
+	if len(r.operandKubernetesVersion) == 0 {
+		return errors.New("missing operand kubernetes version: --operand-kubernetes-version")
+	}
+	if _, err := semver.Parse(r.operandKubernetesVersion); err != nil {
+		return err
+	}
```

Comment:
Could you make this error mention that the bad value came from the `--operand-kubernetes-version` flag?